src/main/java/willy/command/Parser.java

```diff
@@ -0,0 +1,197 @@
+package willy.command;
+
+import willy.exceptions.WillyException;
+import willy.task.*;
+import willy.ui.Greet;
+
+/**
+ * In charge of executing actions based on the user's command.
+ */
+public class Parser {
+    private TaskList list;
+    private static String lastGreeting = "bye";
```

Comment:
Perhaps here the "lastGreeting" variable is a constant and should be named as a constant?

Reply:
I updated the class so the variable is no longer there, but I will take note of such constants, thanks for the suggestion!:)